fluent-package/templates/package-scripts/fluent-package/deb/postinst

```diff
@@ -22,38 +20,113 @@ add_system_user() {
     fi
 }
 
+v4migration=/tmp/.<%= project_name %>_v4migration
+v4migration_with_restart=/tmp/.<%= project_name %>_v4migration_with_restart
+
 add_directories() {
     mkdir -p /var/run/<%= package_dir %>
-    mkdir -p /etc/<%= compat_package_dir %>
-    mkdir -p /etc/<%= compat_package_dir %>/plugin
-    mkdir -p /var/log/<%= compat_package_dir %>
+    mkdir -p /etc/<%= package_dir %>
+    mkdir -p /etc/<%= package_dir %>/plugin
+    mkdir -p /var/log/<%= package_dir %>
+    if [ -d /etc/<%= compat_package_dir %> -a ! -h /etc/<%= compat_package_dir %> ]; then
+	touch $v4migration
+	# /etc/<%= compat_package_dir %> migration from v4
+	if [ -d /etc/<%= compat_package_dir %>/plugin -a -n "$(ls /etc/<%= compat_package_dir %>/plugin)" ]; then
+	    echo "Migrating from /etc/<%= compat_service_name %>/plugin/ to /etc/<%= package_dir %>/plugin/..."
+	    mv -f /etc/<%= compat_service_name %>/plugin/* /etc/<%= package_dir %>/plugin/
+	fi
+	if [ -f /etc/<%= compat_package_dir %>/<%= service_name %>.conf ]; then
+	    # Avoid conflict with packaged version
+	    echo "#"
+	    echo "# Migrating from /etc/<%= compat_package_dir %>/<%= service_name %>.conf to /etc/<%= package_dir %>/<%= service_name %>.conf.rpmmoved"
+	    echo "#"
+	    mv -f /etc/<%= compat_package_dir %>/<%= service_name %>.conf /etc/<%= package_dir %>/<%= service_name %>.conf.dpkg-old
```

Comment:
The echo message wrongly uses `.rpmmoved`.

In the first place, I think we can use a more simple extension, such as `.conf.user`.
I think we can use the same extension between deb and rpm.

Reply:
It is intentional to use such vendor-specific suffix.

about inconsistent message, it should be fixed.